ajax/libs/colors/package.json

```diff
@@ -1,23 +1,36 @@
 {
   "name": "colors",
-  "filename": "colors.min.css",
-  "version": "1.0",
-  "description": "Better colors for the web. A collection of skin classes for faster prototyping and nicer looking sites.",
+  "version": "3.0.0",
+  "filename": "js/colors.min.js",
   "homepage": "http://clrs.cc",
+  "description": "Better default colors for the web. A collection of skin classes for faster prototyping and nicer looking sites.",
   "keywords": [
-    "css",
     "colors",
-    "html",
-    "new"
+    "design",
+    "a11y",
+    "postcss",
+    "palette",
+    "css",
+    "oocss"
   ],
   "repository": {
     "type": "git",
     "url": "https://github.com/mrmrs/colors.git"
   },
-  "license": "MIT",
```

Comment:
Hi @maruilian11 license field moved to [line 25](https://github.com/samme/cdnjs/blob/2aa19cd37abc624dee4739fbde7a541e5936497d/ajax/libs/colors/package.json#L25) but is unchanged.